inst/extdata/meta/metadata.csv

```diff
@@ -0,0 +1,8 @@
+data_type,plot_type,indicator,value_column,indicator_column,indicator_value,name
```

Comment:
Can you add tests for the data in these csvs so that we confirm that wherever an enum is being used that all values correspond to known values? This will affect pretty much all the non-numeric columns.  If numeric data can be validated too that'd be great.

Reply:
Okay I've put some checks in. We're now checking that
* Columns which are enums have values from the expected values
* min and max columns are numeric and min >= 0
* invert_scale is logical
* colour is a character
* there are no NULL, NA or empty cells except in `indicator_column` and `indicator_value` columns (which can be empty)